### PR TITLE
Couple Updates

### PR DESCRIPTION
**Pull request by Dhubbs**
Let me know if this is how I should be doing updates. And if I broke anything

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -168,18 +168,18 @@ <h2 class="font-white thin">ROSTER</h2>
...
-                                <p class="font-gray">Evan Bower</p>
+                                <a href="https://www.instagram.com/dylan_hubble/" class="font-gray">Dylan Hubble</a>
                             </div>
                         </div>
                     </div>
                     <div class="col d-flex p-2">
                         <!--Card-->
                         <div class="card card-cascade">
-                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/brandon.jpeg" alt="Brandon Richardson">
+                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/brandon.jpeg" alt="Ben Begrand">
```
Comment (brandon1024) [inline index.html]: You'll want to switch my picture with the picture for Ben Begrand here.

File: index.html [hunk 2]
```diff
@@ -188,9 +188,9 @@ <h2 class="font-white thin">ROSTER</h2>
                     <div class="col d-flex p-2">
                         <!--Card-->
                         <div class="card card-cascade">
-                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/zane.jpg" alt="Zane Jeffries">
+                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/Chris.jpg" alt="Chris Cashin">
                             <div class="card-body text-center">
-                                <p class="font-gray">Zane Jeffries</p>
+                                <a href="https://www.instagram.com/chr1s_cash1n/" class="font-gray">Chris Cashin</p>
```
Comment (brandon1024) [inline index.html]: Closing tag should be an anchor tag, not a closing paragraph tag:

`<a href="https://www.instagram.com/chr1s_cash1n/" class="font-gray">Chris Cashin</a>`

File: index.html [hunk 3]
```diff
@@ -206,9 +206,9 @@ <h2 class="font-white thin">ROSTER</h2>
                     <div class="col d-flex p-2">
                         <!--Card-->
                         <div class="card card-cascade">
-                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/chelsea.jpg" alt="Chelsea Collicutt">
+                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/chelsea.jpg" alt="Ben Rudderham">
                             <div class="card-body text-center">
-                                <p class="font-gray">Chelsea Collicutt</p>
+                                 <a href="https://www.instagram.com/rudderslam/" class="font-gray">Ben Rudderham</p>
```
Comment (brandon1024) [inline index.html]: Here too:

` <a href="https://www.instagram.com/rudderslam/" class="font-gray">Ben Rudderham</p>`

File: index.html [hunk 4]
```diff
@@ -206,9 +206,9 @@ <h2 class="font-white thin">ROSTER</h2>
                     <div class="col d-flex p-2">
                         <!--Card-->
                         <div class="card card-cascade">
-                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/chelsea.jpg" alt="Chelsea Collicutt">
+                            <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/chelsea.jpg" alt="Ben Rudderham">
```
Comment (brandon1024) [inline index.html]: Should add Ben Rudderhams picture here as well

File: index.html [hunk 5]
```diff
@@ -217,7 +217,7 @@ <h2 class="font-white thin">ROSTER</h2>
                         <div class="card card-cascade">
                             <img class="img-fluid rounded-circle img-responsive roster" src="resources/profiles/dylan.jpg" alt="Dylan Hubble">
```
Comment (brandon1024) [inline index.html]: Should swap your picture with Sarahs

File: index.html [hunk 6]
```diff
@@ -232,89 +232,73 @@ <h2 class="font-gray thin">EXECUTIVE</h2>
...
                         </div>
                         <div class="col-lg-7 col-xl-7">
-                            <h4 class="font-gray">Jessie Bennett</h4>
-                            <h5 class="font-gray">Vice President</h5>
+                            <h4 class="font-gray">Ben Begrand</h4>
+                            <h5 class="font-gray">Director of Operations</h5>
                             <p class="font-gray">
-                                Jessie Bennett is in her second year of Kinesiology. She is a member of the dinghy racing team and occasionally
-                                attends keelboat racing events. She is also a member of the social committee and plans team events throughout the year.
+                                Ben is in his second year of Buisness.He is a member of the dinghy racing team and occasionally
```
Comment (brandon1024) [inline index.html]: Missing whitespace here: `Business.He`

**Comment by Dhubbs** [comment]
Fixed issues and uploaded more pictures

**Comment by brandon1024** [comment]
The images look a bit weird when they are not square, but I'll crop them. Otherwise LGTM! I'll deploy the updates asap

**Comment by Dhubbs** [comment]
Just checking to see if there is an eta for these changes

**Pull request merged**